src/lib/logger.ts

```diff
@@ -5,8 +5,8 @@ import { LaraGlobalType } from "../lara-plugin/index";
 type LoggerEnvironment = "dev" | "production";
 
 const logManagerUrl: Record<LoggerEnvironment, string> = {
-  dev: "//cc-log-manager-dev.herokuapp.com/api/logs",
-  production: "//cc-log-manager.herokuapp.com/api/logs"
+  dev: "//logger.concordqa.org/logs",
+  production: "//logger.concord.org/logs"
```

Comment:
Will http work with these URLs? Perhaps it would be better to hard code them to https?
The only place this will really mater is in dev. If someone is running without https locally I'd guess they will get a bunch of errors or redirects which might or might not save.